pgbot/commands/user.py

```diff
@@ -389,6 +390,27 @@ async def cmd_pet(self):
             "https://raw.githubusercontent.com/PygameCommunityDiscord/"
             + f"PygameCommunityBot/main/assets/images/{fname}",
         )
+        if emotion.pet_anger > common.JUMPSCARE_THRESHOLD:
```

Comment:
Why is this block of code here, under the `pg!pet` section? This is already there under the bonk section lol